tensorflow/lite/kernels/internal/reference/reference_ops.h

```diff
@@ -2789,7 +2789,7 @@ T FloorMod(T input1, T input2) {
                                             std::modulus<T>, FloatMod>::type;
   ModFunc mod_func;
   T trunc_mod = mod_func(input1, input2);
-  return trunc_mod != 0 && (input2 < 0 != trunc_mod < 0) ? trunc_mod + input2
+  return trunc_mod != 0 && ((input2 < 0) != (trunc_mod < 0)) ? trunc_mod + input2
```

Comment:
Might as well put parens around (trunc_mode != 0) and (trunce_mod + input2)

Reply:
@jdduke , thanks for pointing this out, i updated the code as per the review comments, kindly check and approve.

Regards
Amit